### PR TITLE
Added Immovable objects if player collides, changed pressure plates to check block's transform, updated first level colliders (no cameras in place)

### PR DESCRIPTION
**Pull request by varun6531**

### CODE REVIEW

File: ferrous-game/Assets/Immovable.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    private void OnCollisionEnter(Collision c)
+    {
+        if (c.gameObject.tag == "Player")
+        {
+            _rigidbody.isKinematic = true;
+        }
+        
+    }
+
+    private void OnCollisionExit(Collision other)
```
Comment (RuohanLu) [inline ferrous-game/Assets/Immovable.cs:46]: maybe change collision other into collision c?

Comment (varun6531) [reply ferrous-game/Assets/Immovable.cs]: done

**Review by RuohanLu — APPROVED** [review]
LGTM

**Review by RuohanLu — APPROVED** [review]

**Review by edwardsues — APPROVED** [review]

**Comment by edwardsues** [comment]
lgtm!

**Pull request merged**